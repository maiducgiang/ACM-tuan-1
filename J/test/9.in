51
969,154(361)
454,305(474)
662,416(867)
661,981(790)
527,331(551)
951,637(102)
894,870(517)
403,660(947)
140,628(322)
593,957(360)
812,514(620)
449,738(472)
224,385(865)
960,339(656)
199,474(101)
136,120(911)
391,800(585)
470,416(583)
913,141(582)
409,839(274)
726,397(907)
118,146(262)
303,377(696)
683,720(371)
714,946(282)
219,997(495)
231,210(934)
678,207(611)
693,804(108)
513,714(595)
613,808(362)
870,217(965)
393,306(607)
370,434(680)
794,897(228)
284,134(486)
961,941(461)
246,368(249)
178,646(433)
210,307(242)
774,263(586)
272,920(943)
462,846(584)
999,638(898)
445,715(293)
320,454(154)
584,785(675)
615,546(338)
868,781(946)
115,631(927)
327,622(787)
