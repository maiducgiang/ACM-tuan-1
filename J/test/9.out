968185207 999000
151283723 333000
220584817 333000
661319809 999000
26340211 49950
63379031 66600
893975647 999000
403257287 999000
70243847 499500
65929267 111000
405851053 499500
224644367 499500
5604037 24975
959379317 999000
199274627 999000
135984791 999000
8697973 22200
469946167 999000
304076147 333000
81885887 199800
72567151 99900
29507029 249750
101024773 333000
683036651 999000
29759639 41625
12209861 55500
57744931 249750
169382351 249750
3208844 4625
513200881 999000
306597277 499500
217336937 249750
392913301 999000
185032123 499500
264700777 333000
1314122 4625
24024488 24975
82040627 333000
178467787 999000
42019387 199800
773489323 999000
272648023 999000
231191869 499500
49931963 49950
222634789 499500
3201337 9990
2163707 3700
76866349 124875
173582633 199800
1604393 13875
65459033 199800
